### PR TITLE
Reorganize subsections and figures in the seismogram section

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/seismology/seismogram.md [hunk 1]
```diff
@@ -1,51 +1,36 @@
...
+圈层：
 
-因此，地震图包含了震源、结构以及仪器的信息。地震学科研工作者正是通过解读地震图，
-得到相关信息的。一般而言，仪器项（即仪器响应）已知，所以地震图通常用于反演震源和
-结构信息。其中，结构项一般称为格林函数，只与地球内部结构有关。
-
-地震图与元数据（metadata）共同构成了完备的地震学观测数据。
-元数据包含与地震图相关的重要信息，如台站位置、仪器响应等。
+- 地壳（crust）：地球的最表层，大陆地壳的平均厚度为 30-50 km，大洋地壳的平均厚度约为 6 km
+- 地幔（mantle）：地壳以下的固态岩石圈层，最大深度为 2891 km
```
Comment (core-man) [inline source/seismology/seismogram.md:21]:
```suggestion
- 地幔（mantle）：地壳以下的固态岩石层，最大深度为 2891 km
```

File: source/seismology/seismogram.md [hunk 2]
```diff
@@ -1,51 +1,36 @@
...
+- 外核（outer core）：液态铁合金，深度范围为 2891 到 5150 km
```
Comment (core-man) [inline source/seismology/seismogram.md:22]:
```suggestion
- 外核（outer core）：液态铁合金层，深度范围为 2891 到 5150 km
```

File: source/seismology/seismogram.md [hunk 3]
```diff
@@ -1,51 +1,36 @@
...
+- 内核（inner core）：固态铁合金，半径约 1221 km
```
Comment (core-man) [inline source/seismology/seismogram.md:23]:
```suggestion
- 内核（inner core）：固态铁合金层，半径约 1221 km
```

**Comment by seisman** [comment]
Reorganize subsections and figures to make this section more readable. It's not finished and the content need to be polished, but I think at least the current structure makes more sense.

**Review by core-man — APPROVED** [review]

**Pull request merged**